### DISCUSSION TITLE
Ability to set focal point on images

### DISCUSSION THREAD

**Discussion by goellner** [filamentphp/filament#14652] [Ideas]
It would be awesome to get a button to the image editor, where we can set the focal point on images. When using  a CDN like imgix or twicpics, we can set a focal point and the image gets cropped with the center set on that specific point. 

Statamic has a very good UI for this, attached a video of it:

https://github.com/user-attachments/assets/78da3088-d6ee-42f3-834a-be11250fbd74

We can set X,Y,Z

where X and Y are values from 0 to 1
Z is a value from 1 to 10

CDNs want different formats, but that could be easily transformed from those values. Some want 34%, some 0.34, ...

And the media library package from Spatie also has support for focal cropping: https://spatie.be/docs/image/v3/image-manipulations/resizing-images#content-focal-crop

**Comment by Noaber** [comment]
Setting a focal point would be awesome!
